Exercise/Parse/PlayerData/Program.cs

```diff
@@ -0,0 +1,131 @@
+﻿using System;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+public class PlayerData
+{
+    public string Username { get; set; }
+    public int Level { get; set; }
+    public string Guild { get; set; }
+    public List<string> Items { get; set; }
+    public int SkinId { get; set; }
+
+    public PlayerData()
+    {
+        Items = new List<string>();
+    }
+
+    public PlayerData(string username, int level, string guild, List<string> items, int skinId )
+    {
+        Username = username;
+        Level = level;
+        Guild = guild;
+        Items = items;
+        SkinId = skinId;
+     
+
+    }
+    
+
+}
+
+public class JsonParser
+{
+    public static PlayerData ParsePlayerDataFromJson(string jsonData)
+    {
+        try
+        {
+            PlayerData playerData = JsonConvert.DeserializeObject<PlayerData>(jsonData);
+            return playerData;
+        }
+        catch (JsonException ex)
+        {
+           
+            Console.WriteLine($"Error parsing JSON: {ex.Message}");
+            return null;
+        }
+    }
+}
+
+
+class Program
+{
+    static void Main(string[] args)
+    {
+        while (true)
+        {
+            Console.WriteLine("Choose a Number (1, 2, 3) or enter 'q' to quit:");
+            string choice = Console.ReadLine();
+
+            if (choice == "q")
+            {
+                break; 
+            }
+
+            switch (choice)
+            {
+                case "1":
+                    DisplayExample(1);
+                    break;
+                case "2":
+                    DisplayExample(2);
+                    break;
+                case "3":
+                    DisplayExample(3);
+                    break;
+                default:
+                    Console.WriteLine("Invalid choice. Please enter 1, 2, 3, or 'q' to quit.");
+                    break;
+            }
+        }
+    }
+
+    static void DisplayExample(int exampleNumber)
+    {
+        string jsonData = GetJsonDataForExample(exampleNumber);
+        if (jsonData != null)
```

Comment:
too much null check is happening, think of other ways to make sure no null exceptions happen